genie-common/src/main/java/com/netflix/genie/common/dto/JobStatusMessage.java

```diff
@@ -67,6 +67,39 @@
     public static final String JOB_PROCESS_NOT_FOUND = "Couldn't check job process status.";
 
     /**
+     * Job PID terminated, exist status is yet to be verified.
+     */
+    public static final String PROCESS_DETECTED_TO_BE_COMPLETE = "Process detected to be complete";
```

Comment:
Agree that's a better name. 
Are we going to cherry-pick this in `master`? (I'd say yes, but not completely sure what's the difference between the two right now)
Because if not, I don't think we should rename, as it would make cherry-picks a pain.